google-startup-scripts/usr/lib/systemd/system/google-shutdown-scripts.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Google Compute Engine user shutdown scripts.
+After=local-fs.target network-online.target network.target
+After=google.service rsyslog.service
+Wants=local-fs.target network-online.target network.target
+
+[Service]
+ExecStart=/bin/true
```

Comment:
This should be:
ExecStart=/usr/share/google/run-shutdown-scripts

And you don't need ExecStop at all. This job is to run the script- it should run when the job starts not when it stops.